app/decorators/calendar_decorator.rb

```diff
@@ -0,0 +1,18 @@
+class CalendarDecorator < Draper::Decorator
+  delegate_all
+
+  def render_destroy_link
+    if h.current_user.position.present? &&
+      (h.current_user.position == Position.find_by(name: User::SECRETARY))
+      h.link_to "remove", h.calendar_path(object),
+        { method: :delete, class: 'btn btn-danger btn-xs' }
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Redundant curly braces around a hash parameter.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.